ThoR/Alloy/Elab.lean

```diff
@@ -312,8 +319,34 @@ private def createRelationAliasCommands
   : List ((TSyntax `command)) := Unhygienic.run do

```

Comment:
Can createRelationAliasCommands and createSignatureAliasCommands be merged into one function?